src/components/Header.jsx

```diff
@@ -1,33 +1,42 @@
 import React from "react";
+import { Trans } from "react-i18next";
 import "../styles/headerPage.scss";
 import headerImages from "../helpers/bannerImages.js";
 
-export default function header() {
+export default function Header() {
   return (
     <div className="header-box">
       <div className="header-items-box">
         <div className="header-logo-box">
-          <img src={headerImages.meubelHouseLogo}></img>
-          <img src={headerImages.skinClinicLogo}></img>
+          <img src={headerImages.meubelHouseLogo} alt="Meubel House" />
+          <img src={headerImages.skinClinicLogo} alt="Skin Clinic" />
         </div>
         <div className="header-text-box">
-          <a>Home</a>
-          <a>Shop</a>
-          <a>About</a>
-          <a>Contact</a>
+          <a>
+            <Trans i18nKey="nav.home">Home</Trans>
+          </a>
+          <a>
+            <Trans i18nKey="nav.shop">Shop</Trans>
+          </a>
+          <a>
+            <Trans i18nKey="nav.about">About</Trans>
+          </a>
+          <a>
+            <Trans i18nKey="nav.contact">Contact</Trans>
+          </a>
         </div>
         <div className="header-icons-box">
           <a>
```

Comment:
If you are not using the anchor element with a link or so, maybe this element should be transformed on a another more semantic HTML

Reply:
In the future, I plan to have these elements take you to their corresponding page when clicked.